### ISSUE TITLE
virtualbox: linux-kernel-5.3.8 cannot install virtualbox module

### ISSUE THREAD

**Issue by eatradish** [AOSC-Dev/aosc-os-abbs#2042]
Bug description
-----------------------

linux-kernel-5.3.8 cannot install virtualbox module

Update channel(s)
--------------------------

Testing-proposed, Testing, Stable-Proposed, Stable

Package and version
------------------------------
```
➜  blog dpkg -s linux-kernel-5.3.8 
Package: linux-kernel-5.3.8
Status: install ok installed
Section: kernel
Installed-Size: 189444
Maintainer: Mingcong Bai <jeffbai@aosc.io>
Architecture: amd64
Version: 5.3.8-1
Description: Linux Kernel version 5.3.8 for desktop usage
➜  blog dpkg -s virtualbox 
Package: virtualbox
Status: install ok installed
Section: emulation
Installed-Size: 188980
Maintainer: Mingcong Bai <jeffbai@aosc.io>
Architecture: amd64
Version: 6.0.10-1
Depends: acpica-unix (>= 20180629), cdrkit (>= 1:1.1.11-1), libidl2 (>= 0.8.14-2), openssl (>= 1.1.1c), curl (>= 7.65.3), gcc (>= 8.3.1), libpng (>= 1.6.37), python-2 (>= 2.7.16-3), sdl-ttf (>= 2.0.11-4), libvpx (>= 1.7.0-1), libxml2 (>= 2.9.9-2), procps (>= 3.3.15), shared-mime-info (>= 1.10), zlib (>= 1.2.11), qt-5 (>= 1:5.12.2+wk5.212.0-1), vde2 (>= 1:2.3.2-1), net-tools (>= 20160828), dkms (>= 1:2.6.1), make (>= 4.2.1), linux-pam (>= 1.3.1), xorg-server (>= 1.20.3-1), x11-app (>= 7.7.20180328)
Description: Full-featured virtualization software suite from Oracle
```

**Comment by KexyBiscuit** [comment]
Upstream fix: https://www.virtualbox.org/ticket/18911
Update VirtualBox to ^6.0.12 should fix this.

**Comment by eatradish** [comment]
Fix on 7faafac5eceb26ee03dda0f78ec64d39f96de1c8, But have not yet pushed to the stable branch
